### ISSUE TITLE
Add ability to give record edit access to another Zenodo user

### ISSUE THREAD

**Issue by chrisfilo** [zenodo/zenodo#1325]
I would like to share edit access to a record with other package maintainers. I could not find this functionality.

**Comment by krzysztof** [comment]
Hey, it's not there yet, but it's a popular feature request.
We don't know when we'll have enough time to work on it yet, since it involves a lot of considerations all over Zenodo once we start thinking about multiple-owners of records, although something simpler like allowing community members to edit the metadata of accepted records might be a first step.

Relevant:
https://github.com/zenodo/zenodo/issues/151
https://github.com/zenodo/zenodo/issues/35

**Comment by oesteban** [comment]
Hi, any news on this?

**Comment by jonquet** [comment]
Is this addressed in Zenodo now ?
Documentation for this ?

**Comment by xJREB** [comment]
> Is this addressed in Zenodo now ? Documentation for this ?

@jonquet, yes, it's possible now. There is a "Share" button in the top-right corner.
